### PR TITLE
Update `OBAM 2022` to Group stage 

### PR DESCRIPTION
**Pull request by Laphii1206**
- add Group stage mappool
 - add Participants 
 - update Group stage format  from groups 5 to groups of 4
 - update Tournament Dates.

## Self-check

- [x] The changes are tested against the [contribution checklist](https://osu.ppy.sh/wiki/osu!_wiki/Contribution_guide#self-check)

### CODE REVIEW

**Comment by TicClick** [comment]
if you are editing original articles (a list of tournaments, in this case) in a way that adds substantially new content, you have to either make same changes to the translations, or properly outdate them, so no, this doesn't warrant the `SKIP_OUTDATED_CHECK` tag

**Comment by Laphii1206** [comment]
oh i see

File: wiki/Tournaments/o!BAM/2/en.md [hunk 1]
```diff
@@ -48,8 +48,84 @@ The osu! Battle Archive Malaysia 2022 is run by various community members.
 - [Discord server](https://discord.gg/GYqWsQbEbF)
 - [Livestream](https://www.twitch.tv/osumys)
 
+## Participants
+
+| Seed No. | Team | Members | Z-Sum |
+| :-: | :-- | :-- | :-- |
+| 1 | **sturrige curry** | ::{ flag=MY }:: **[DuoX](https://osu.ppy.sh/users/9560694)**, ::{ flag=ID }:: [i love uma](https://osu.ppy.sh/users/7108275) | 16.43 |
+| 2 | **Anti-Cunny Action** | ::{ flag=MY }:: **[not_aweeb](https://osu.ppy.sh/users/9375317)**, ::{ flag=ID }:: [Rexeez](https://osu.ppy.sh/users/1987591) | 15.26 |
```
Comment (TPGPL) [inline wiki/Tournaments/o!BAM/2/en.md:56]:
```suggestion
| 2 | **Anti-Cunny Action** | ::{ flag=MY }:: **[not\_aweeb](https://osu.ppy.sh/users/9375317)**, ::{ flag=ID }:: [Rexeez](https://osu.ppy.sh/users/1987591) | 15.26 |
```

File: wiki/Tournaments/o!BAM/2/en.md [hunk 2]
```diff
@@ -48,8 +48,84 @@ The osu! Battle Archive Malaysia 2022 is run by various community members.
...
+| 6 | **Miyu Archive** | ::{ flag=MY }:: **[Houshou Marine](https://osu.ppy.sh/users/6402211)**, ::{ flag=ID }:: [-Reuto-](https://osu.ppy.sh/users/10717635) | 12.44 |
+| 7 | **RUDOLF ENJOYERS** | ::{ flag=MY }:: **[Rampax](https://osu.ppy.sh/users/3995630)**, ::{ flag=SG }:: [Inquisitives](https://osu.ppy.sh/users/10722794) | 11.76 |
+| 8 | **Cat Enjoyer** | ::{ flag=MY }:: **[Lunasa](https://osu.ppy.sh/users/16436446)**, ::{ flag=SG }:: [GSBlank](https://osu.ppy.sh/users/2312106) | 11.49 |
+| 9 | **Where Love Lies** | ::{ flag=MY }:: **[Tokino Sora](https://osu.ppy.sh/users/6996926)**, ::{ flag=ID }:: [Fuma](https://osu.ppy.sh/users/1501956) | 10.39 |
+| 10 | **The Cunnysseurs** | ::{ flag=MY }:: **[Blethz](https://osu.ppy.sh/users/11918602)**, ::{ flag=ID }:: [BlankTap](https://osu.ppy.sh/users/10137131) | 5.88 |
+| 11 | **Flover** | ::{ flag=MY }:: **[heyimcrunchy](https://osu.ppy.sh/users/13067221)**, ::{ flag=SG }:: [Hecatia](https://osu.ppy.sh/users/8244635) | 4.65 |
+| 12 | **osu retirement** | ::{ flag=MY }:: **[iraozm](https://osu.ppy.sh/users/12443104)**, ::{ flag=TH }:: [_Arigato](https://osu.ppy.sh/users/9317938) | 4.57 |
```
Comment (TPGPL) [inline wiki/Tournaments/o!BAM/2/en.md:66]:
```suggestion
| 12 | **osu retirement** | ::{ flag=MY }:: **[iraozm](https://osu.ppy.sh/users/12443104)**, ::{ flag=TH }:: [\_Arigato](https://osu.ppy.sh/users/9317938) | 4.57 |
```

File: wiki/Tournaments/o!BAM/2/en.md [hunk 3]
```diff
@@ -48,8 +48,84 @@ The osu! Battle Archive Malaysia 2022 is run by various community members.
...
+| 13 | **yes seggs** | ::{ flag=MY }:: **[Chiyuu](https://osu.ppy.sh/users/8226107)**, ::{ flag=MY }:: [Zygody](https://osu.ppy.sh/users/3677251) | 3.57 |
+| 14 | **Dreamy Color** | ::{ flag=MY }:: **[Jolene](https://osu.ppy.sh/users/13626098)**, ::{ flag=MY }:: [Chibi Maruko](https://osu.ppy.sh/users/5585377) | 3.40 |
+| 15 | **Vanity** | ::{ flag=MY }:: **[Inugami Korone](https://osu.ppy.sh/users/4474918)**, ::{ flag=MY }:: [Mumei Nanashi](https://osu.ppy.sh/users/5414124) | 1.87 |
+| 16 | **chungus capoo** | ::{ flag=MY }:: **[GKen](https://osu.ppy.sh/users/12633052)**, ::{ flag=VN }:: [realshin](https://osu.ppy.sh/users/8006029) | 0.52 |
+| 17 | **openureyesyouwillsee** | ::{ flag=MY }:: **[Judicious](https://osu.ppy.sh/users/8087710)**, ::{ flag=SG }:: [_gt](https://osu.ppy.sh/users/8301957) | -1.19 |
```
Comment (TPGPL) [inline wiki/Tournaments/o!BAM/2/en.md:71]:
```suggestion
| 17 | **openureyesyouwillsee** | ::{ flag=MY }:: **[Judicious](https://osu.ppy.sh/users/8087710)**, ::{ flag=SG }:: [\_gt](https://osu.ppy.sh/users/8301957) | -1.19 |
```

File: wiki/Tournaments/o!BAM/2/en.md [hunk 4]
```diff
@@ -48,8 +48,84 @@ The osu! Battle Archive Malaysia 2022 is run by various community members.
...
+| 18 | **NAJIB FANCLUB** | ::{ flag=MY }:: **[Hu_Ai](https://osu.ppy.sh/users/10910261)**, ::{ flag=SG }:: [JokThree](https://osu.ppy.sh/users/7713152) | -1.49 |
+| 19 | **Offensize Name** | ::{ flag=MY }:: **[rhatson](https://osu.ppy.sh/users/14826877)**, ::{ flag=TH }:: [heathenz_](https://osu.ppy.sh/users/14180969) | -1.51 |
```
Comment (TPGPL) [inline wiki/Tournaments/o!BAM/2/en.md:73]:
```suggestion
| 19 | **Offensize Name** | ::{ flag=MY }:: **[rhatson](https://osu.ppy.sh/users/14826877)**, ::{ flag=TH }:: [heathenz\_](https://osu.ppy.sh/users/14180969) | -1.51 |
```

File: wiki/Tournaments/o!BAM/2/en.md [hunk 5]
```diff
@@ -48,8 +48,84 @@ The osu! Battle Archive Malaysia 2022 is run by various community members.
...
+| 27 | **Japanese Goblin** | ::{ flag=MY }:: **[Mano Aloe](https://osu.ppy.sh/users/5429513)**, ::{ flag=MY }:: [Watson Amelia](https://osu.ppy.sh/users/12674638) | -4.63 |
+| 28 | **Damn bratty crewmate** | ::{ flag=MY }:: **[Floth](https://osu.ppy.sh/users/11134301)**, ::{ flag=MY }:: [Axorite](https://osu.ppy.sh/users/9124278) | -4.91 |
+| 29 | **Ikan** | ::{ flag=MY }:: **[-quack](https://osu.ppy.sh/users/16502529)**, ::{ flag=MY }:: [Izayoii](https://osu.ppy.sh/users/15542056) | -6.96 |
+| 30 | **Mandekissafriend** | ::{ flag=MY }:: **[Ookami Mion](https://osu.ppy.sh/users/8009988)**, ::{ flag=MY }:: [Hachune Miku](https://osu.ppy.sh/users/10584780) | -7.7 |
+| 31 | **Marv and Willy show** | ::{ flag=MY }:: **[Its not Marvin](https://osu.ppy.sh/users/13600612)**, ::{ flag=MY }:: [Hanami-](https://osu.ppy.sh/users/8703188) | -8.03 |
+| 32 | **AMOGUS SUSSY BAKA S_** | ::{ flag=MY }:: **[Lollicornzi](https://osu.ppy.sh/users/14454744)**, ::{ flag=MY }:: [cherry sweet](https://osu.ppy.sh/users/12079158) | -8.05 |
```
Comment (TPGPL) [inline wiki/Tournaments/o!BAM/2/en.md:86]:
```suggestion
| 32 | **AMOGUS SUSSY BAKA S\_** | ::{ flag=MY }:: **[Lollicornzi](https://osu.ppy.sh/users/14454744)**, ::{ flag=MY }:: [cherry sweet](https://osu.ppy.sh/users/12079158) | -8.05 |
```

File: wiki/Tournaments/o!BAM/2/en.md [hunk 6]
```diff
@@ -48,8 +48,84 @@ The osu! Battle Archive Malaysia 2022 is run by various community members.
...
+## Groups
+
+| Group | Top Seed | High Seed | Low Seed | Unseeded |
+| :-: | :-- | :-- | :-- | :-- |
+| **A** | kinda hot tho | Flover | :froggy: | 洪家派驰名海鲜煮炒餐馆 |
+| **B** | ENERGY GAP | Dreamy Color | LENG ZAI | Damn bratty crewmate |
+| **C** | Miyu Archive | chungus capoo | Offensize Name | Marv and Willy show |
+| **D** | RUDOLF ENJOYERS | yes seggs | koko soko asoko | Can We Get Much Higher |
+| **E** | Diggas in paris | The Cunnysseurs | openureyesyouwillsee | Japanese Goblin |
+| **F** | Cat Enjoyer | Vanity | NAJIB FANCLUB | AMOGUS SUSSY BAKA S_ |
```
Comment (TPGPL) [inline wiki/Tournaments/o!BAM/2/en.md:97]:
```suggestion
| **F** | Cat Enjoyer | Vanity | NAJIB FANCLUB | AMOGUS SUSSY BAKA S\_ |
```

File: wiki/Tournaments/o!BAM/2/en.md [hunk 7]
```diff
@@ -48,8 +48,84 @@ The osu! Battle Archive Malaysia 2022 is run by various community members.
...
 ## Mappools
 
+### Group stage
+
+**[Download the mappack here! (77 MB)](https://cdn.discordapp.com/attachments/959526009546346516/1018855345138962494/OBAM2_Groupstage.rar)**
+
+- NoMod
+  1. [Feryquitous feat. Aitsuki Nakuru - Bascor (Seolv) \[Glorious Collab\]](https://osu.ppy.sh/beatmapsets/1090138#osu/2278971)
+  2. [Roselia - Charles (Yugu) \[kanor's Rua^\]](https://osu.ppy.sh/beatmapsets/839553#osu/1758680)
+  3. [Hatsuki Yura - Halloween of the Dead 2(Elayue) \[Happy Halloween\]](https://osu.ppy.sh/beatmapsets/1542855#osu/3153693)
```
Comment (TPGPL) [inline wiki/Tournaments/o!BAM/2/en.md:110]:
```suggestion
  3. [Hatsuki Yura - Halloween of the Dead 2 (Elayue) \[Happy Halloween\]](https://osu.ppy.sh/beatmapsets/1542855#osu/3153693)
```

File: wiki/Tournaments/o!BAM/2/en.md [hunk 8]
```diff
@@ -48,8 +48,84 @@ The osu! Battle Archive Malaysia 2022 is run by various community members.
...
+  4. [Mitsukiyo - Hifumi Daisuki (Amateurre) \[Extraordinary\]](https://osu.ppy.sh/beatmapsets/1829914#osu/3755787)
+  5. [brz1128 vs. M-UE - Bloody Marquis(Athanasia-) \[Athanasia's Extra\]](https://osu.ppy.sh/beatmapsets/1783848#osu/3653240)
+  6. [KAF feat. KAFU - phony(Astrolis) \[Sayonara\]](https://osu.ppy.sh/beatmapsets/1654426#osu/3376899)
```
Comment (TPGPL) [inline wiki/Tournaments/o!BAM/2/en.md:113]:
```suggestion
  5. [brz1128 vs. M-UE - Bloody Marquis (Athanasia-) \[Athanasia's Extra\]](https://osu.ppy.sh/beatmapsets/1783848#osu/3653240)
  6. [KAF feat. KAFU - phony (Astrolis) \[Sayonara\]](https://osu.ppy.sh/beatmapsets/1654426#osu/3376899)
```

File: wiki/Tournaments/o!BAM/2/en.md [hunk 9]
```diff
@@ -48,8 +48,84 @@ The osu! Battle Archive Malaysia 2022 is run by various community members.
...
+- Hidden
+  1. [xi - Parousia (Niva) \[Eschatology (AR8.5)\]](https://osu.ppy.sh/beatmapsets/1713530#osu/3501365)
+- HardRock
+  1. [Mifune Shioriko (CV: Koizumi Moeka) - Aoi Canaria (Murada-) \[True Freedom\]](https://osu.ppy.sh/beatmapsets/1706527#osu/3487001)
+  2. [solfa - Yami no Rinne (feat. yuki)(LMT) \[Allure of Darkness\]](https://osu.ppy.sh/beatmapsets/1739716#osu/3555764)
```
Comment (TPGPL) [inline wiki/Tournaments/o!BAM/2/en.md:118]:
```suggestion
  2. [solfa - Yami no Rinne (feat. yuki) (LMT) \[Allure of Darkness\]](https://osu.ppy.sh/beatmapsets/1739716#osu/3555764)
```

File: wiki/Tournaments/o!BAM/2/en.md [hunk 10]
```diff
@@ -48,8 +48,84 @@ The osu! Battle Archive Malaysia 2022 is run by various community members.
...
+- DoubleTime
+  1. [Harumaki Gohan - Saikai (AirinCat) \[Fujirin's Insane\]](https://osu.ppy.sh/beatmapsets/1602654#osu/3312899)
+  2. [NIKITA - ELECTRiCiTY (Rokkea) \[Another\]](https://osu.ppy.sh/beatmapsets/241372#osu/640771)
+  3. [lical - nyctalopia (Raijodo) \[hard\]](https://osu.ppy.sh/beatmapsets/1284666#osu/2766955)
+- DoubleTime + Hidden
+  1. [Yellow Zebra - Orange Diz (Frill) \[gow's Insane\]](https://osu.ppy.sh/beatmapsets/17306#osu/73514)
+- FreeMod
+  1. [DJ Sharpnel - Marunouchi survivor (ampzz) \[GALV'S SENRYAKU!!\]](https://osu.ppy.sh/beatmapsets/113330#osu/294015)
+  2. [yuki - Clammbon (Short Ver.)(Garden) \[Insane\]](https://osu.ppy.sh/beatmapsets/1707693#osu/3489464)
```
Comment (TPGPL) [inline wiki/Tournaments/o!BAM/2/en.md:127]:
```suggestion
  2. [yuki - Clammbon (Short Ver.) (Garden) \[Insane\]](https://osu.ppy.sh/beatmapsets/1707693#osu/3489464)
```

**Review by TPGPL — CHANGES_REQUESTED** [review]

**Review by TPGPL — APPROVED** [review]

**Pull request merged**